util/getSkills.test.js

```diff
@@ -0,0 +1,60 @@
+import { getSkills } from './getSkills';
+
+describe('getSKills', () => {
+
+  it('returns an arry of strings',  () => {
+    const testData = {
+      certifications: [
+        "CPWA",
+        "AWS"
+      ],
+      technologies: "JavaScript, Elvish",
+      expertise: "Mentorship, swords, bows, horses",
+      strengths: "Leading teams, Fighting monsters",
+      interests: "Camping every weekend, singing, playing guitar",
+    }
+    const skills = getSkills(testData);
+    expect(Array.isArray(skills));
+  });
+
+  it('transforms a csv value into an array of strings', () => {
+    const testData = {
+      expertise: "Mentorship, swords, bows, horses",
+    };
+    const result = getSkills(testData);
+    expect(Array.isArray(result));
+  });
```

Comment:
I love tests!! LOL

In this test are we not testing the same output as the test above? (duplicate)